### ISSUE TITLE
Parenthesis for max should include min term

### ISSUE THREAD

**Issue by redone17** [KeiraZhao/MDAN#1]
https://github.com/KeiraZhao/MDAN/blob/4eaec5b0d49a3af446b3b52850f646dfed507a14/main_amazon.py#L143

Would it rather be:

`loss = torch.max(losses + mu * torch.min(domain_losses))`

to correspond with Eq(5) of https://papers.nips.cc/paper/8075-adversarial-multiple-source-domain-adaptation.pdf?
